fix: attach binaries to node manager release

jacderida opened the pull request:
The crate name was specified as `sn_node_manager` rather than `sn-node-manager`.

Also disable the `sn_service_management` crate from being released on Github. At the same time I arranged the crates in the config file alphabetically, just because it makes them a bit easier to find.

## Description

reviewpad:summary

davidrusu commented:
Thanks for the pointer, switched the format to follow the env logger default, it seems like a good starting point.

I've also pushed a change to include the spans after the module path. I've updated the OP comment to show the examples

jacderida committed:
fix: attach binaries to node manager release

The crate name was specified as `sn_node_manager` rather than `sn-node-manager`.

Also disable the `sn_service_management` crate from being released on Github. At the same time I
arranged the crates in the config file alphabetically, just because it makes them a bit easier to
find.
